js/modules/loki_message_api.js

```diff
@@ -92,7 +94,22 @@ class LokiMessageAPI {
       };
 
       try {
-        await rpc(`http://${nodeUrl}`, this.snodeServerPort, 'store', params);
+        const result = await rpc(
+          `http://${nodeUrl}`,
+          this.snodeServerPort,
+          'store',
+          params
+        );
+
+        // Make sure we aren't doing too much PoW
+        const currentDifficulty = window.storage.get('PoWDifficulty', null);
+        const newDifficulty = result.difficulty;
+        if (
+          !Number.isNaN(newDifficulty) &&
```

Comment:
Just wondering if `result.difficulty` is a string or a number?

Reply:
Number